File: Beecrowd/1165 Número Primo/main.py
def EPrimo(pX):
    for i in range(2, pX):
        if pX % i == 0:
            return 0
    return 1


N = int(input())

for i in range(0, N):
    X = int(input())
    if EPrimo(X):
        print(f"{X} eh primo")
    else:
        print(f"{X} nao eh primo")

File: Beecrowd/1165 Número Primo/main.cpp
#include <iostream>

using namespace std;

bool ePrimo(int pX);

int main()
{
    int x, n;

    cin >> n;

    for (int i = 0; i < n; i++)
    {
        cin >> x;
        if (ePrimo(x))
            cout << x << " eh primo" << endl;
        else
            cout << x << " nao eh primo" << endl;
    }

    return 0;
}

bool ePrimo(int pX)
{
    for (int i = 2; i < pX; i++)
    {
        if (pX % i == 0)
            return false;
    }
    return true;
}
